_docs/principles/accessibility.md

```diff
@@ -0,0 +1,216 @@
+
+{% figure [caption: "Little spacing and unclear relationships"] [class:"image bg-light border"] %}
+![]({{site.urll}}/assets/images/overview/accessibility/accessibility-space-1.svg)
+{% endfigure %}
+
+{% endcolumn %}
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+{% figure [caption: "More spacing and clear relationships"] [class:"image bg-light border"] %}
+![]({{site.urll}}/assets/images/overview/accessibility/accessibility-space-2.svg)
+{% endfigure %}
+
+{% endcolumn %}
+
+</div>
+
+
+<div class="hxRow">
+
+{% column left:"hxCol-12 hxCol-xs-12 hxCol-sm-12 hxCol-md-12 hxCol-lg-12" %}
+
+### Provide easily identifiable feedback
+
+Provide feedback for interactions, such as confirming form submission, alerting the user when something goes wrong, or notifying the user of changes on the page. Ensure instructions are easy to identify and suggestions to help fix issues whenever errors are returned must be provided. Present any important feedback that requires user action in a prominent style.
+
+{% endcolumn %}
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+#### Failed text inputs
+
+There are many acceptable ways to make text inputs visually accessible. If you’re indicating a failed text input, make sure to include multiple visual indicators. For example, in addition to the error message, the border color turns red and doubles in thickness from 1px to 2px to provide distinguishable visual queues not reliant on color.
+
+It is important to reserve the prominent border at 2px for failed text input as an additional visual indicator for error state.
+
+{% endcolumn %}
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+{% figure [class:"image bg-light border"] %}
+![caption:"Default state"]({{site.url}}/assets/images/overview/accessibility/accessibility-inputs-1.svg)
+{% endfigure %}
+
+{% endcolumn %}
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+{% figure [class:"image bg-light border"] %}
+![caption:"Error state"]({{site.url}}/assets/images/overview/accessibility/accessibility-inputs-2.svg)
+{% endfigure %}
+
+{% endcolumn %}
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+#### Focus for Input Field
+
+In this example, we provide a clear visual feedback that the focus has moved to the input field by changing the border color and adding a highlight to give the user clear indication of where they’re currently taking action.
+
+{% endcolumn %}
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+{% figure [class:"image bg-light border"] %}
+![caption:"Default state"]({{site.url}}/assets/images/overview/accessibility/accessibility-inputs-3.svg)
+{% endfigure %}
+
+{% endcolumn %}
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+{% figure [class:"image bg-light border"] %}
```

Comment:
Also, should this be error state as well?

Reply:
One of the states was incorrect. I updated the caption on that one from error to focused.